### PR TITLE
Send chunks after biomes changed

### LINKED ISSUE #372: Biomes not working 100%
Greetings

**Describe the bug**
- When creating a Skyblock Island, the biome is different than the one put in the schematics.yml file. After regenerating, it seems to generate the biome correctly (not always).

**To Reproduce**
1. Create an Island.
2. Check the Island's biome.
(3. Regen the Island.)
(4. Check the Island's biome.)

**Expected behavior**
- The biomes should be the same as the ones put in the schematics.yml file upon creating the Island.

** Server Version and Plugin Version**
Copy and paste the results of `/version` and `/is about`

- Version: This server is running Paper version git-Paper-771 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
You are 15 version(s) behind
Previous version: git-Paper-"ae892dc" (MC: 1.16.5)

- Plugin version; 3.0.8 (pinned version in support channel on Discord).

**Screenshots**
- Isn't really necessary concerning this issue in my opinion.

**Logs**
If there are error messages in your console, please send them via [this](https://paste.gg/)
- There are no errors.

**Additional context**
- I have explained everything already.

Thanks in advance
Milan

### PR DESCRIPTION
**Pull request by MertUnverdi**
Fixed #372 
Thank you intellij i've edited this on web cause intellij doesn't commits changes (my intellij broken)

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java [hunk 1]
```diff
@@ -101,10 +101,11 @@ public void setIslandBiome(@NotNull Island island, @NotNull XBiome xBiome) {
         getIslandChunks(island, world).thenAccept(chunks -> {
             Location pos1 = island.getPos1(world);
             Location pos2 = island.getPos2(world);
-            xBiome.setBiome(pos1, pos2);
-            for (Chunk chunk : chunks) {
-                IridiumSkyblock.getInstance().getNms().sendChunk(world.getPlayers(), chunk);
-            }
+            xBiome.setBiome(pos1, pos2).thenAccept(unused -> {
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java:104]: use .thenRun instead of thenAccept

Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java:104]: Sure? XSeries runs async

Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java:104]: ? wdym thenRun just removes the unused variable

**Comment by dlsf** [comment]
This needs to be tested. I once did this or something very similar and it either didn't work or break.

**Comment by MertUnverdi** [comment]
i tried on 1.16.5 its worked for me and #372 owner

**Review by dlsf — APPROVED** [review]

**Pull request merged**